applications/MultilevelMonteCarloApplication/tests/test_MultilevelMonteCarloApplication_mpi.py

```diff
@@ -0,0 +1,52 @@
+# Import Kratos
+import KratosMultiphysics
+import KratosMultiphysics.MultilevelMonteCarloApplication
+try:
+    import KratosMultiphysics.mpi as KratosMPI
+except ImportError:
+    raise Exception("KratosMPI could not be imported!")
```

Comment:
```suggestion
if not KM.IsDistributedRun():
    raise Exception("This test script can only be executed in MPI!")
```

check this file, it is up to date: https://github.com/KratosMultiphysics/Kratos/blob/master/applications/CoSimulationApplication/tests/test_CoSimulationApplication_mpi.py